worlds/sc2/Items.py

```diff
@@ -8,12 +10,13 @@
 class ItemData(typing.NamedTuple):
     code: typing.Optional[int]
     type: str
-    number: int
+    number: int  # Important for bot commands to send the item into the game
     race: SC2Race
     classification: ItemClassification = ItemClassification.useful
     quantity: int = 1
     parent_item: typing.Optional[str] = None
     origin: typing.Set[str] = {"wol"}
+    description: str = None
```

Comment:
Type here should be `Optional[str]` or should have no default (or maybe default to empty string). Otherwise mypy yells at you.